<ui version="4.0">
 <class>VistaRegistro</class>
 <widget class="QMainWindow" name="VistaRegistro">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>500</width><height>600</height></rect>
  </property>
  <property name="windowTitle">
   <string>MenULE - Registro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_nombre">
    <property name="geometry"><rect><x>150</x><y>60</y><width>200</width><height>30</height></rect></property>
    <property name="placeholderText">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_apellido">
    <property name="geometry"><rect><x>150</x><y>100</y><width>200</width><height>30</height></rect></property>
    <property name="placeholderText">
     <string>Apellido</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_correo">
    <property name="geometry"><rect><x>150</x><y>140</y><width>200</width><height>30</height></rect></property>
    <property name="placeholderText">
     <string>Correo</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry"><rect><x>150</x><y>180</y><width>200</width><height>30</height></rect></property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contrase√±a</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_rol">
    <property name="geometry"><rect><x>150</x><y>220</y><width>200</width><height>30</height></rect></property>
    <item><property name="text"><string>estudiante</string></property></item>
    <item><property name="text"><string>profesor</string></property></item>
    <item><property name="text"><string>personal_comedor</string></property></item>
    <item><property name="text"><string>visitante</string></property></item>
    <item><property name="text"><string>administrador</string></property></item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_tui">
    <property name="geometry"><rect><x>150</x><y>260</y><width>200</width><height>30</height></rect></property>
    <property name="placeholderText">
     <string>TUI (solo estudiantes)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_registrar">
    <property name="geometry"><rect><x>180</x><y>310</y><width>120</width><height>30</height></rect></property>
    <property name="text">
     <string>Registrar</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
